### PR TITLE
etcd: return ttlExpiryTime in GetResponse

### PR DESCRIPTION
**Pull request by JoshVanL**
See https://github.com/dapr/components-contrib/pull/2863

### CODE REVIEW

File: state/etcd/etcd.go [hunk 1]
```diff
@@ -141,9 +144,25 @@ func (e *Etcd) Get(ctx context.Context, req *state.GetRequest) (*state.GetRespon
 		return &state.GetResponse{}, nil
 	}
 
+	var metadata map[string]string
+	if resp.Kvs[0].Lease > 0 {
+		leaseResp, err := e.client.TimeToLive(ctx, clientv3.LeaseID(resp.Kvs[0].Lease))
```
Comment (ItalyPaleAle) [inline state/etcd/etcd.go:149]: Does this make a network call?

Comment (JoshVanL) [reply state/etcd/etcd.go:149]: Yes, but there is no other way around this.

Comment (ItalyPaleAle) [reply state/etcd/etcd.go:149]: I am concerned the perf impact in this case is not insigificant if we need to make an extra roundtrip to the database

Comment (JoshVanL) [reply state/etcd/etcd.go:149]: Indeed, I agree. It's also not even taking the now() clock time of the etcd process.

**Comment by ItalyPaleAle** [comment]
Can this be closed in favor of #2932?

**Comment by JoshVanL** [comment]
@ItalyPaleAle yeah I'm happy to do that if you don't mind the review :slightly_smiling_face: 

I'll update now.

/close

**Pull request closed without merging**